OpenIPSL/Examples/Tutorial/Example_4/BaseModels/BaseNetwork/SMIBPartial.mo

```diff
@@ -0,0 +1,100 @@
+    B=0.0000) annotation (Placement(transformation(extent={{28,-30},{48,-10}})));
+  inner OpenIPSL.Electrical.SystemBase SysData
+    annotation (Placement(transformation(extent={{-100,80},{-60,100}})));
+  GeneratingUnits.InfiniteBus infiniteBus(
+    P_0=pf.powerflow.machine.PG2,
+    Q_0=pf.powerflow.machine.QG2,
+    v_0=pf.powerflow.bus.v2,
+    angle_0=pf.powerflow.bus.A2,          displayPF=true)
+    annotation (Placement(transformation(extent={{94,-10},{74,10}})));
+  replaceable
+  OpenIPSL.Electrical.Loads.PSSE.Load load(
+    P_0=pf.powerflow.load.PL1,
+    Q_0=pf.powerflow.load.QL1,
+    v_0=pf.powerflow.bus.v3,
+    angle_0=pf.powerflow.bus.A3) constrainedby OpenIPSL.Electrical.Loads.PSSE.BaseClasses.baseLoad
+    annotation (Placement(transformation(extent={{-34,-56},{-14,-36}})));
+  OpenIPSL.Electrical.Events.PwFault pwFault(
+    R=0.01,
+    X=0.1,
+    t1=1,
+    t2=1.1) annotation (Placement(transformation(
+        extent={{-6,-6},{6,6}},
+        rotation=270,
+        origin={36,-46})));
+  PFData.PowerFlow pf(redeclare record PowerFlow =
+        OpenIPSL.Examples.Tutorial.Example_4.PFData.PF00000)
+    annotation (Placement(transformation(extent={{-80,26},{-60,46}})));
+equation
+  connect(line_01.p, B01.p)
+    annotation (Line(points={{-53,0},{-60,0}}, color={0,0,255}));
+  connect(line_01.n, B03.p)
+    annotation (Line(points={{-35,0},{-30,0}}, color={0,0,255}));
+  connect(line_02.p, B03.p) annotation (Line(points={{11,20},{-16,20},{-16,0},{
+          -30,0}}, color={0,0,255}));
+  connect(line_02.n, B02.p)
+    annotation (Line(points={{29,20},{48,20},{48,0},{60,0}}, color={0,0,255}));
+  connect(line_03.p, B03.p) annotation (Line(points={{-7,-20},{-16,-20},{-16,0},
+          {-30,0}}, color={0,0,255}));
+  connect(line_03.n, B04.p)
+    annotation (Line(points={{11,-20},{20,-20}},color={0,0,255}));
+  connect(line_04.p, B04.p)
+    annotation (Line(points={{29,-20},{20,-20}}, color={0,0,255}));
+  connect(line_04.n, B02.p) annotation (Line(points={{47,-20},{48,-20},{48,0},{
+          60,0}}, color={0,0,255}));
+  connect(infiniteBus.pwPin, B02.p)
+    annotation (Line(points={{73,0},{60,0}}, color={0,0,255}));
+  connect(load.p, B03.p)
+    annotation (Line(points={{-24,-36},{-24,0},{-30,0}}, color={0,0,255}));
+  connect(pwFault.p, B04.p) annotation (Line(points={{36,-39},{36,-28},{24,
+          -28},{24,-20},{20,-20}}, color={0,0,255}));
+  annotation (Icon(coordinateSystem(preserveAspectRatio=false), graphics={
+          Rectangle(extent={{-100,100},{100,-100}}, lineColor={135,135,135})}),
+                                                                 Diagram(
+        coordinateSystem(preserveAspectRatio=false), graphics={
+                                                Text(
+          extent={{-92,68},{94,48}},
+          lineColor={0,0,0},
+          lineThickness=1,
+          fontSize=15,
+          textStyle={TextStyle.Bold},
+          textString="Single-machine infinite bus model*")}));
```

Comment:
Just out of curiosity, is that a particular reason for the "*" after "...infinite bus model"? Just wanted to make sure it is itentional.

Reply:
There was a reason when I created the first version of that model during my MSC studies. I will remove the character because I don't remember why I put it there.

Reply:
I know exactly where that asterisk comes from ... copy and paste from `OpenIPSL.Examples.KundurSMIB.SMIB` where the asterisk points to the P. Kundur book as footnote ;-) 
So it is fine to remove it.